src/Api/Dto/TaskPackageDto.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace AppBundle\Api\Dto;
+
+use Symfony\Component\Serializer\Annotation\Groups;
+
+class TaskPackageDto
+{
+    #[Groups(["task"])]
+    public readonly string $short_code;
+
+    #[Groups(["task"])]
+    public readonly string $name;
+
+    #[Groups(["task"])]
+    public readonly string $type;
+
+    #[Groups(["task"])]
+    public readonly int $volume_per_package;
+
+    #[Groups(["task"])]
+    public readonly int $quantity;
+
+    public function __construct(
+        string $shortCode,
+        string $name,
+        int $averageVolumeUnits,
+        int $quantity)
+    {
+        $this->short_code = $shortCode;
+        $this->name = $name;
+        //FIXME; why do we have name and type with the same value?
```

Comment:
it is legacy, i don't think it is used in the rider app anymore?

i spotted one usage here coopcycle-app/src/navigation/store/NewDeliveryForm.js in the source